src/sass/components/_product.scss

```
@@ -0,0 +1,87 @@
+.product {
+
+  &__container {
+    margin-top: 25rem;
+  }
+
+  &__image {
+    border: 1px solid #ccc;
+    position: relative;
+    cursor: pointer;
+
+    &:hover {
+      color: $orange;
+      transition: all 0.5s ease-out;
+
+      .product__info {
+        @include pd($x: 0.5rem);
+        @include flex($justify: space-evenly);
+        @include font($fs:$fs-light);
+
+        .icon {
+          padding-right:0.5rem;
+          color: $white;
+
+          &:hover {
+            color: $orange;
+            transition: all 0.5s ease-out;
+          }
+
+          i {
+            margin-right:0.5rem;
+          }
+        }
+      }
+    }
+  }
+
+  &__tag {
+    position:absolute;
+    width: 5rem;
+    top: 0;
+    left: 0;
+
+    img {
+      height: 100%;
+    }
+  }
+  &__info {
+    @include flex($direction: row, $align: center, $justify: center, $gap:2rem);
+    display: none;
+    position:absolute;
+    height:50px;
```

```suggestion
    position: absolute;
    height: 50px;
```
hiện tại có đang xài prettier hay eslint gì k Tuấn ơi

---

cài cả 2 nhưng mà có nhiều lúc nó ko tự format ấy